src/Shmuelie.WinRTServer/CsWinRT/ComServerExtensions.cs

```diff
@@ -83,4 +83,16 @@ public static bool UnregisterClassFactory(this ComServer server, BaseClassFactor
 
         return server.UnregisterClassFactory(factory.Clsid);
     }
+
+    /// <summary>
+    /// Get or create a CCW using the default ComWrappers object used by this extension class.
+    /// </summary>
+    /// <param name="managedObject">Managed object to create a CCW for.</param>
+    /// <exception cref="ArgumentNullException">The managed object being passed is null.</exception>
+    public static nint GetOrCreateComInterfaceForObject(object managedObject)

```

Comment:
@shmuelie do you want me to add it to `strategyBased` extensions too? Do you want me to add it to `winrtserverExtensiosn` too?

Reply:
done